app/src/main/res/values/strings.xml

```diff
@@ -10,196 +10,196 @@
+    <string name="http_server">HTTP server</string>
+    <string name="udp_exporter">UDP exporter collector</string>
     <string name="no_dump">None</string>
     <string name="no_dump_info">PCAP will not be dumped</string>
     <string name="http_server_info">Start an HTTP server for the PCAP download</string>
     <string name="udp_exporter_info">Sends the PCAP to a remote UDP receiver</string>
-    <string name="http_server_port">HTTP Server Port</string>
-    <string name="receiver_ip_address">Collector IP Address</string>
-    <string name="receiver_port">Collector Port</string>
-    <string name="dump_mode">Dump Mode</string>
-    <string name="user_guide">User Guide</string>
-    <string name="open_telegram_group">Telegram Group</string>
-    <string name="source_code">Source Code</string>
-    <string name="yes">YES</string>
-    <string name="no">NO</string>
-    <string name="existing_vpn_confirm">The currently active VPN app will be disconnected. Do you want to proceed?</string>
-    <string name="vpn_setup_failed">VPN setup failed</string>
-    <string name="app_not_found">App "%1$s" not found</string>
-    <string name="apps_loading_please_wait">Apps loading in progress, please wait</string>
+    <string name="http_server_port">Port</string>
+    <string name="receiver_ip_address">IP address</string>
+    <string name="receiver_port">Port</string>
+    <string name="dump_mode">Dumping mode</string>
+    <string name="user_guide">User guide</string>
+    <string name="open_telegram_group">Telegram group</string>
+    <string name="source_code">Copylefted libre source code</string>
+    <string name="yes">Yes</string>
+    <string name="no">No</string>
+    <string name="existing_vpn_confirm">Disconnect the active VPN app and continue?</string>
+    <string name="vpn_setup_failed">Could not set up VPN</string>
+    <string name="app_not_found">Could not find the \"%1$s\" app</string>
+    <string name="apps_loading_please_wait">Loading apps…</string>
     <string name="stats">Stats</string>
-    <string name="active_connections">Active Connections</string>
-    <string name="dropped_connections">Dropped Connections</string>
-    <string name="tot_connections">Total Connections</string>
-    <string name="open_sockets">Open Sockets</string>
+    <string name="active_connections">Active connections</string>
+    <string name="dropped_connections">Dropped connections</string>
+    <string name="tot_connections">Total connections</string>
+    <string name="open_sockets">Open sockets</string>
     <string name="max_file_descriptor">Max FD</string>
-    <string name="bytes_sent">Bytes Sent</string>
-    <string name="bytes_rcvd">Bytes Received</string>
-    <string name="packets_sent">Packets Sent</string>
-    <string name="packets_rcvd">Packets Received</string>
-    <string name="dns_queries">DNS Queries</string>
-    <string name="search_apps">Search Apps</string>
+    <string name="bytes_sent">Bytes sent</string>
```

Comment:
Could be below "Bytes" and "Packets" menus, but that is another matter

Reply:
I'm aware some translators expect strings to be within their context, but I adopt the convention to append new localization strings at the bottom of the file to help translators identify them.